go/vt/topo/conn.go

```diff
@@ -113,6 +113,13 @@ type Conn interface {
 	// Returns ErrInterrupted if ctx is canceled.
 	Lock(ctx context.Context, dirPath, contents string) (LockDescriptor, error)
 
+	// TryLock provides exactly same functionality as 'Lock', the only difference is
+	// it tires its best to be non-blocking call. Non-blocking is the best effort though. Depends
+	// on implementation it might result in blocking client (for some scenario) just like any other
```

Comment:
```suggestion
	// on the implementation it may result in the client being blocked in some scenarios just like any other
```

Reply:
We are saying "some scenarios". Do we know what scenarios those are?

Reply:
change the wordings a bit.